content/en/apps/reference/app-settings/token_login.md

```diff
@@ -0,0 +1,31 @@
+---
+title: ".token_login"
+linkTitle: ".token_login"
+weight: 5
+description: >
+  **Token login**: Instructions and schema for Login by SMS
+relatedContent: >
+  apps/reference/api#login-by-sms
+ 
+keywords: sms login users
+---
+
+Login via SMS settings are defined under the `token_login` key, as an object supporting the following properties:
+## `app_settings.json .sms`
+| property         | type | required       | description                                                                                                                                                                              |
+|------------------|------|---------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| enabled | Boolean | yes | Enables or disables token_login deployment-wide. When this is false, users can't be updated to use token_login and any requests to login with a token link will fail.  |
+| app_url | String | yes | Defined the base URL of the app. |
+| translation_key | String | yes if `message` is not defined | Translation key for the information (helper) sms message that the user receives, along with they token-login link |
+| message | String | yes if `translation_key` is not defined | Message content for the information (helper) sms message that the user receives, along with they token-login link |
+
+## Code sample
+
+The definition takes the typical form below:
+
+```json
+"token_login": {
+  "app_url": "https://your-deployment-url.com/",
```

Comment:
Is there ever a time where the app_url would be different for token_login than the rest of the site? Is it so that a URL shortener could be specified for this feature? If not, than this seems like a setting that we'd want at the top level of app_settings, and only required if using a feature that needs it (like this one).

Reply:
We can't use a url shortener that we don't host, because that would mean passing all of our "free login" links to a 3rd party. 
I can move it to the root of app settings!

Reply:
Also, I _could_ get the host in express (by using `req.hostname` http://expressjs.com/en/5x/api.html#req.hostname ). Unsure if the host would always be correct when the request passes through load balancing. cc @garethbowen

Reply:
Yeah I'd leave it configurable to be safe.

Reply:
@abbyad If I move `app_url` to root level of `app_settings`, how would you suggest I document it being required for this feature? Would simply mentioning it suffice?

Reply:
Ya, I think having it at the root level doc mentioning it as required if `token_login` specified, and cross referencing that in the `token_login` works. That would be my preference.

Reply:
Thanks for your input!